expyfun/stimuli/_crm.py

```diff
@@ -178,7 +180,7 @@ def crm_prepare_corpus(fs, path_out=None, overwrite=False, dtype=np.float64,
         options.
     overwrite : bool
         Whether or not to overwrite the files that may already exist in
-        ``path_out``.
+        ``path_out``. Can also be 'skip' to skip any existing files.
```

Comment:
It would be nice, but remember how seldom this is going to be run. For most people: once, ever. With that in mind, it just didn't seem worth it. And if one is re-running it, it's likely because the resampling or something was changed, in which case you would want to replace them all. So it just didn't seem worth it for the added coding time, complexity, and increased probability of screwing something up.

Other changes look good. Yes, I can add an example.